src/components/OSCALControlProse.test.js

```diff
@@ -59,24 +59,64 @@ function proseParamValuesEditingRenderer() {
 }
 
 function testOSCALControlProseEditing(parentElementName, renderer) {
-  test(`${parentElementName} displays description and parameter inputs`, async () => {
+  beforeEach(() => {
     renderer();
     screen
       .getByRole("button", {
         name: "edit-bycomponent-component-1-statement-control-1_smt.a",
       })
       .click();
+  });
 
-    const descriptionTextField = screen.getByLabelText("Description");
+  renderer();
+  screen
+    .getByRole("button", {
+      name: "edit-bycomponent-component-1-statement-control-1_smt.a",
+    })
+    .click();
+  const paramValueTextField = screen.getByLabelText("control-1_prm_1");
+  const descriptionTextField = screen.getByLabelText("Description");
+  const changeDescription = () =>
+    fireEvent.change(descriptionTextField, {
+      target: { value: "Test Description" },
+    });
+  const saveChanges = () =>
+    screen
+      .getByRole("button", {
+        name: "save-control-1_smt.a",
+      })
+      .click();
```

Comment:
Please see https://github.com/EasyDynamics/oscal-react-library/pull/529#discussion_r934536462 which I apparently submitted just a moment too late.

Reply:
That makes sense! My most recent commit has reverted the tests back to their previous commit.

Reply:
Thanks! Really appreciate your flexibility and @tuckerzp and I will sync a bit on testing practices to get a little more consensus here.